cookbook/controller/error_pages.rst

```diff
@@ -250,6 +250,127 @@ This approach allows you to create centralized and layered error handling:
 instead of catching (and handling) the same exceptions in various controllers
 time and again, you can have just one (or several) listeners deal with them.
 
+A centralized error handling that also allows to log exceptions might look like this::
+
+    // src/AppBundle/EventListener/AcmeExceptionListener.php
+    namespace AppBundle\EventListener;
+
+    use Psr\Log\LoggerInterface;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+    use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+
+    class AcmeExceptionListener
```

Comment:
Now that we have a dedicated AppBundle, we no longer need to use "Acme". Let's just name this `ExceptionListener`.